### PR TITLE
Adds command line part 2 content

### PR DESCRIPTION
**Pull request by eelanagaraj**
Please let me know if you have any thoughts/feedback, for instance if you think of additional terms that would be helpful to include in the course content!

### CODE REVIEW

File: content/docs/command/command_line_part2.md [hunk 1]
```diff
@@ -13,3 +13,181 @@ description = "Goals"
...
+| Navigate file system and working with files & folders (demo) | 10-15 minutes        | ❌                                |
+| Quick overview of package managers                           | 5-10 minutes         | ❌                                |
+| Class break                                                  | 10 minutes           | ❌                                |
+| Command line practice in groups                              | 40-50 minutes        | local terminal OR repl.it account |
+| Summary & Next class overview                                | 5 minutes            | ❌                                |
+
+## Useful Terms/Concepts
+
+- **File**: data, usually organized according to some specific _file format_ (e.g. `txt`, `mp3`, `docx`)
+- **Directory**: a file that stores information about the length and location of other files, similar to a table of contents
```
Comment (marcshaw) [inline content/docs/command/command_line_part2.md:33]: Would it be more

> a folder that stores files or other directories

🤔

File: content/docs/command/command_line_part2.md [hunk 2]
```diff
@@ -13,3 +13,181 @@ description = "Goals"
...
+| `echo`  | print some text                                                                                                                                                                                                                        |
+| `cat`   | print the contents of the file                                                                                                                                                                                                         |
+| `>`     | this is an operator: `X > Y` will redirect (send) the outputs of command `X` to `Y`; if `Y` exists, it this will _overwrite_ the existing file.                                                                                        |
+| `>>`    | this is an operator similar to the above: `X >> Y` will send the outputs of command `X` to `Y`; if `Y` exists, it this will _append_ to the existing file.                                                                             |
+
```
Comment (cbenjemaa) [inline content/docs/command/command_line_part2.md:64]: I think we should point out that we can create a script file where we group some commands to achieve certain automation.

File: content/docs/command/command_line_part2.md (hunk 1 above)
Comment (eelanagaraj) [reply content/docs/command/command_line_part2.md:33]: will add that as well, but technically in linux-systems directories are just special files -- one of the videos they watched talks about this a bit so hopefully it'll tie to that a bit

File: content/docs/command/command_line_part2.md (hunk 2 above)
Comment (eelanagaraj) [reply content/docs/command/command_line_part2.md:64]: will add a note but also mention this in the demo section

**Pull request merged**